### ISSUE TITLE
DaprController.java as a non-springboot controller

### ISSUE THREAD

**Issue by hadeed85** [dapr/java-sdk#976]
Could we make this a non-springboot controller in case we want to use the Actor SDK without springboot? Also, can we also have this available as a gRPC endpoint at some point?

https://github.com/dapr/java-sdk/blob/master/sdk-springboot/src/main/java/io/dapr/springboot/DaprController.java

**Comment by artursouza** [comment]
That is an interesting request. It should be possible to create a sdk-spring artifact and move spring-only classes there.

**Comment by yaron2** [comment]
> Hi, Can i take-up this task.

cc @artursouza

**Comment by artursouza** [comment]
Yes. Assigning to you now. Ping me if you need more context. I think that creating a spring project is better here. Ideally this is a non-breaking change and any class that will be deleted should first be marked as deprecated.
